physics/GWD/drag_suite.F90

```diff
@@ -1363,7 +1363,8 @@ subroutine drag_suite_run(                                           &
             DO k=kts,km
                wsp=SQRT(uwnd1(i,k)**2 + vwnd1(i,k)**2)
                ! alpha*beta*Cmd*Ccorr*2.109 = 12.*1.*0.005*0.6*2.109 = 0.0759
-               var_temp = 0.0759*EXP(-(zl(i,k)/H_efold)**1.5)*a2*          &
+               ! Change alpha to 35 -- 0.0759 becomes 0.2214
+               var_temp = 0.2214*EXP(-(zl(i,k)/H_efold)**1.5)*a2*          &
```

Comment:
@haiqinli @mdtoyNOAA Will this change affect global and HAFS applications as well ?  Have any tests been done to check the impact on applicaitons other than RRFS ?

Reply:
We tested this change in the HR3 C768 FV3GFS and detected improvements to the 500hPa ACC scores.

Reply:
Mike, that is good know.  I'd assume this has major impact on surface winds and turbulent exchange coefficients as well.  Did you also see positive impact on grid-to-obs stats ?

Reply:
Mike, since this change is hardwired and will be include in HR4.  Please record this change in our tracking spreadsheet.

Reply:
@yangfanglin Keep in mind that this PR is just targeting the RRFS.v1 branch.

Reply:
@yangfanglin It may be included in a future PR targeting the main repo, but that will be a separate, future step.

Reply:
> Mike, that is good know. I'd assume this has major impact on surface winds and turbulent exchange coefficients as well. Did you also see positive impact on grid-to-obs stats ?

In the 10m winds we see significant improvements in RMSE in both west and east CONUS.  However, a slight degradation of windspeed bias in west (little change in bias in east).